lib/actions.js

```diff
@@ -84,8 +84,15 @@ exports.ensureEmptyDirectory = function() {
   if(fs.existsSync(this.destinationRoot())) {
     // A valid directory won't have any files/folders.
     var files = fs.readdirSync(this.destinationRoot());
+    var file;
     if (files.length > 0) {
-      this.env.error(chalk.red(this.destinationRoot(), 'is not an empty directory'));
+      var count;
```

Comment:
Is this part still necessary now that writing of the SDK files occurs later? I would prefer this check actually checks for an empty directory.